node/stakeValuesProc/stakeValuesProcessor.go

```diff
@@ -1,102 +1,97 @@
-package totalStakedAPI
+package stakeValuesProc
```

Comment:
`stakevalueprocessor` 
- no camel case in the package name
- no abbreviations

Reply:
done.